### ISSUE TITLE
Cannot start server_box on MacOS Monterey 12.6.8

### ISSUE THREAD

**Issue by galabl** [lollipopkit/flutter_server_box#182]
**Describe the bug**
Cannot open app, more info below.

**To Reproduce**
1. Download the latest MacOS release.
2. Click to open the app
3. The error is shown

**Desired Results**
Running app

**Actual Results**
3 different errors

**Screenshots**
![image](https://i.imgur.com/7aE9quV.png)

**Device**
- OS: MacOS Monterey
- App Version: 1.0.493

**Comment by lollipopkit** [comment]
This is may caused by Apple's security policy, you can try `sudo xattr -d com.apple.quarantine DOWNLOADED_APP_PATH`.

I will distribute it on the App Store in the future.

**Comment by galabl** [comment]
@lollipopkit I tried both 
`sudo xattr -d com.apple.quarantine server_box.app/` and 
`sudo xattr -d com.apple.quarantine server_box.app/Contents/MacOS/server_box` and it didn't help, these are some errors from the report that I can see, not sure if it's helpful.

```
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Incident Identifier: 1EFA62A1-A00F-4E6C-9337-44C68D9DDD2B
CrashReporter Key:   D3ECC034-E075-2DA1-5AC8-779BA7A60CBD
Hardware Model:      MacBookPro11,4
Process:             server_box [5563]
Path:                /Users/USER/Downloads/*/server_box.app/Contents/MacOS/server_box
Identifier:          tech.lolli.serverBox
Version:             1.0.551 (551)
Code Type:           X86-64 (Native)
Role:                Default
Parent Process:      launchd [1]
Coalition:           tech.lolli.serverBox [842]

Date/Time:           2023-10-13 10:07:32.7373 +0200
Launch Time:         2023-10-13 10:07:32.7255 +0200
OS Version:          macOS 12.6.8 (21G725)
Release Type:        User
Report Version:      104

Exception Type:  EXC_CRASH (SIGKILL (Code Signature Invalid))
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY
Termination Reason: CODESIGNING 1 

Triggered by Thread:  0

Thread 0 Crashed:
0                                	       0x10e45bae0 _dyld_start + 0
1   ???                           	       0x10237d000 ???


Thread 0 crashed with X86 Thread State (64-bit):
  rax: 0x0000000000000000  rbx: 0x0000000000000000  rcx: 0x0000000000000000  rdx: 0x0000000000000000
  rdi: 0x0000000000000000  rsi: 0x0000000000000000  rbp: 0x0000000000000000  rsp: 0x00007ff7bdb825c8
   r8: 0x0000000000000000   r9: 0x0000000000000000  r10: 0x0000000000000000  r11: 0x0000000000000000
  r12: 0x0000000000000000  r13: 0x0000000000000000  r14: 0x0000000000000000  r15: 0x0000000000000000
  rip: 0x000000010e45bae0  rfl: 0x0000000000000200  cr2: 0x0000000000000000
  
Logical CPU:     0
Error Code:      0x00000000 
Trap Number:     0


Binary Images:
       0x10e457000 -        0x10e4c2fff  (*) <baab2c6d-3797-3264-8595-a591561979de> 
       0x10237d000 -        0x102380fff ??? (*) <062a9392-969a-348d-85e2-004d6a16e432> ???

Error Formulating Crash Report:
dyld_process_snapshot_get_shared_cache failed

EOF


```

**Comment by lollipopkit** [comment]
![image](https://github.com/lollipopkit/flutter_server_box/assets/10864310/a2019ed4-80e4-4980-ad4a-43aeda776062)
It's signing error. It seems that i must upload it to AppStore...

**Comment by galabl** [comment]
@lollipopkit hmm, I don't think that's the case, maybe resign and rebuild the app would work, I ran bunch off apps on my Mac outside app store.

Also, maybe better option would be to add it to homebrew instead of app store https://docs.brew.sh/Adding-Software-to-Homebrew

**Comment by lollipopkit** [comment]
[server_box.app.zip](https://github.com/lollipopkit/flutter_server_box/files/12891292/server_box.app.zip)
Try this?

**Comment by galabl** [comment]
> [server_box.app.zip](https://github.com/lollipopkit/flutter_server_box/files/12891292/server_box.app.zip) Try this?

Same issue

**Comment by lollipopkit** [comment]
https://apps.apple.com/app/id1586449703

Now available on AppStore.

**Comment by galabl** [comment]
@lollipopkit I just get an empty window after installing from AppStore
![empty-screen](https://github.com/lollipopkit/flutter_server_box/assets/13868892/b48646a9-5114-46d4-a59b-9c14dc5331f9)

**Comment by lollipopkit** [comment]
I don't have this problem after downloading from the App Store....
No problem both m1 and x86...
I don't know what to do for now.

**Comment by galabl** [comment]
> I don't have this problem after downloading from the App Store.... No problem both m1 and x86... I don't know what to do for now.

Mine is intel one, do you have any idea how could I check some logs or something?

Do you know can I build my own version without paid developer account, maybe that would give more info?

**Comment by lollipopkit** [comment]
Fixed in 1.0.717, sorry about that.
